mullvad.js

```diff
@@ -18,16 +18,13 @@ const DEFAULT_ITEMS = {
     type: {name: _('VPN Type'), text: '', gSetting: 'show-type'},
 };
 
+const IO_ERROR = 7;
```

Comment:
Why declare the const here when it's provided by `Soup.KnownStatusCode.IO_ERROR`? I generally prefer to track the library's const.

Reply:
[Soup.KnownStatusCode](https://valadoc.org/libsoup-2.4/Soup.KnownStatusCode.html) is deprecated since 2.44, but we should probably just be using [Soup.Status.IO_ERROR](https://valadoc.org/libsoup-2.4/Soup.Status.IO_ERROR.html) instead rather than defining the const ourselves.

Reply:
I agree that using a library defined value would be better, but `IO_ERROR` doesn't seem to exist in Soup 3.0 anymore since since it isn't listed here: https://gjs-docs.gnome.org/soup30~3.0/soup.status

Reply:
Oops, I pushed your changed but modified it to use Soup.Status.IO_ERROR. You may be correct on this one, if it doesn't exist I'll have to restore your original change -- sorry for my stupidity :upside_down_face:

Reply:
You're right, it appears to be undefined (I forgot how much of a pain getting output from a Gnome extension is, took me a bit to remember how.)

I'm going to restore your change now in the main branch, thanks for pointing it out.